README.md

```diff
@@ -44,23 +50,23 @@ module mymodule
 go 1.21.0
 
 require (
-	github.com/optimizely/go-sdk v2.0.0-beta
+	github.com/optimizely/go-sdk/v2 v2.0.0
```

Comment:
/v2 is physical folder?  wondering why we need both /v2 and v2.0.0 :)

Reply:
We don't need a physical folder for v2. Master branch will hold the latest major version. There will be a release branch as well for future releases of v2.x.x or even v1.x.x. We have already released/tagged branches which are serving previously released versions.

We both need /v2 & v2.0.0 to avoid module path collision for v2 & v1. With this, user can use both versions in the same project. For example:

```
import (
     sdkv1 github.com/optimizely/go-sdk
     sdkv2 github.com/optimizely/go-sdk/v2
)

func main(){
     clientv1 := sdkv1.Client()
     clientv2 := sdkv2.Client()
}
```

and in user's application module:
```
require (
     github.com/optimizely/go-sdk v1.8.5
     github.com/optimizely/go-sdk/v2 v2.0.0
)
```
If we don't have that `/v2` suffix, then the user has no way to use both the versions. Though users hardly use two versions in same project, this is the convention for library code in Go to follow.